lib/WebpackOptionsApply.js

```diff
@@ -59,6 +59,8 @@ const NamedChunksPlugin = require("./NamedChunksPlugin");
 const DefinePlugin = require("./DefinePlugin");
 const SizeLimitsPlugin = require("./performance/SizeLimitsPlugin");
 
+const StatsPlugin = require("./stats/index");
```

Comment:
Dunno if you need to include index here, `require` will pick it up. At least does for us at webpack-cli.

Reply:
Yups, will change it. :smile:

Reply:
Using multiple `require` would be easier to maintain. Moreover, `StatsPlugin` suggests that it returns a _plugin_, which is not the case.